parlai/tasks/dialogue_safety/agents.py

```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Dialogue safety related datasets and teachers."""
+
+import parlai.core.build_data as build_data
+from parlai.core.message import Message
+from parlai.core.teachers import FixedDialogTeacher
+
+import copy
+import json
+import os
+import random
+import sys as _sys
+
+try:
+    import pandas as pd
+except ImportError:
+    raise ImportError('Please install pandas by running `pip install pandas`')
+
+
+# Constants
+OK_CLASS = '__ok__'
+NOT_OK_CLASS = '__notok__'
+
+SINGLE_TURN_DATA = 'single_turn_safety.json'
+MULTI_TURN_DATA = 'multi_turn_safety.json'
+
+
+def build(datapath):
```

Comment:
guess we're doing this now instead of a separate `build.py` file?

Reply:
i don't really know. what do you prefer?

Reply:
i don't personally care, though maybe good to move to a `build.py` file as that is what we specify in the official docs to do (so might as well keep that consistent)